lambdaclass/mirra_backend
# [GH-723] Add Consumable items Configuration

## Motivation

Migrate items from the `config.json` file to a relational database. 

Closes #723 

## Summary of changes

- Although we call them `items` these items are more likely to be `consumable items` since they only exists during a match. So, I've added the `ConsumableItem` MVC 
- Added a LiveView form for consumable items as a basis for further implementations
- The new config for items has been integrated into the game, you'll see some changes in arena's logic.
- We will only send to the client items that are active
DISCLAIMER: We still have pending to add the relationship between items and effects, but that's going to be addressed in another issue #747 . At the moment it is just a map displayed in the view

## How to test it?

Go to `localhost:4100/consumable_items` modify or add current items (be careful, our client could crash because we have to add the needed assets for a new item) and then play a match and check that everything works!.

## Checklist
